docs/developer/services/explorer_service.rst

```diff
@@ -93,18 +90,53 @@ The explorer API provides the following endpoints.
 
   Retrieve the metadata of a node with ``id``. Valid for all nodes.
 
+**GET /graph/node/:id/edges**                    
+
+  ``GenericExplorerController.retrieveNodeEdges( id: Long )``
+
+  Get all in- and outgoing edges of a node with  ``id``.
+
+
+**GET /graph/nodes/:prop**                       
```

Comment:
should these be done in the form of query parameters rather then URL? e.g. /graph/nodes?prop=woop

Same goes for the ones below. It's fine for me to leave these as is for the moment if that's how they're implemented in the explorer, but they will likely need to be changed during the API refactor.

Reply:
For now I had just updated the docs with the current state of the API. I didn't look into the refactoring yet